src/main/java/uk/gov/hmcts/reform/sscs/personalisation/Personalisation.java

```diff
@@ -136,15 +136,21 @@ private String getName(SubscriptionType subscriptionType, SscsCaseData ccdRespon
 
         if (subscriptionType.equals(APPELLANT)
                 && ccdResponse.getAppeal().getAppellant() != null) {
-            name = ccdResponse.getAppeal().getAppellant().getName();
+            return getDefaultName(ccdResponse.getAppeal().getAppellant().getName(), "");
         } else if (subscriptionType.equals(REPRESENTATIVE)
                 && hasRepresentative(wrapper)) {
-            name = ccdResponse.getAppeal().getRep().getName();
+            return getDefaultName(ccdResponse.getAppeal().getRep().getName(), "Sir / Madam");
```

Comment:
See if you can use the existing uk.gov.hmcts.reform.sscs.service.SendNotificationService#getRepSalutation method.

Reply:
Had a stab at this. I made a small change to your code to enable this so please re-review.